2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -386,11 +344,7 @@ cl_nc_correlations %>%
 # Create a unified peptide sequence to protein assignment
 new_seq_to_master <- pep_infiles %>%
   camprotR::get_parsimony_pep2prot()
-#> With original assignments, 1951 master proteins. With update, 1820 master proteins
```

Comment:
This information is output to console so shouldn't be any need to hard-code it in as comment lines